### ISSUE TITLE
[Suggestion] Startup flag for new computers

### ISSUE THREAD

**Issue by BenNottelling** [bmrf/tron#155]
Hi there, I think Tron is a really great script to use, and it's something I like to run on new PCs to make sure they are a bit cleaner than what would come out of the box, I'm also using it with my job to remove some things from school laptops such as the Xbox apps (because it's completely unnecessary to have)
One thing which I think would be a great addition is a flag for new PCs so Tron skips some things such as  Anti-Malware & Anti-Virus scans, debloating 3rd part apps, cleaning cache, creating a system restore point, ect. Basically anything that's not going to be needed on a new PC

**Comment by qwertychouskie** [comment]
Agreed.  Tron is a great way of getting rid of the garbage on new systems, especially OEM systems with all their bloatware.

**Comment by BenNottelling** [comment]
That's for the suggestion, but I feel like that's not as helpful as a flag for a new computer as that'll still leave out windows optimizations and anti spyware patches, which is why I felt like it would be helpful as a flag

**Comment by vocatus** [comment]
In that case, running the Stage 2 and Stage 4 scripts should do everything you're looking for (bloat removal, telemetry disabling, etc). Tron doesn't really do any optimizations to Windows outside of a couple _very_ minor tweaks; the real meat of it is in Stages 2 and 4.
